src/math/monte_carlo_integration.jl

```diff
@@ -0,0 +1,20 @@
+"""
+Monte carlo integration is a very easy and scalable way to do multidimentional integrals.
+However, only single variable integrals is considered.
+
+
+This function takes three outputs:
```

Comment:
Please add proper documentation like in the [`knapsack`](https://github.com/TheAlgorithms/Julia/blob/main/src/knapsack/knapsack.jl) problem.

Reply:
Done in the monte carlo PR